assets/controllers/form_reveal_controller.js

```diff
@@ -46,4 +46,14 @@ export default class extends Controller {
             this.buttonTarget.hidden = false;
         }
     }
+
+    openByValue(event) {
+        this.sectionTargets.forEach((element)=>{
+            if(element.dataset.value === event.target.value) {
+                element.hidden = false;
+            }else {
+                element.hidden = true;
+            }
+        })
```

Comment:
(Détail) On n'a pas (encore) de formatteur pour la partie JS / CSS mais voici comment Prettier le formatterait

```suggestion
        this.sectionTargets.forEach((element) => {
            if (element.dataset.value === event.target.value) {
                element.hidden = false;
            } else {
                element.hidden = true;
            }
        });
```